templates/repo/home.tmpl

```diff
@@ -10,7 +10,7 @@
 		<div class="ui secondary menu">
 			{{if .PullRequestCtx.Allowed}}
 				<div class="fitted item">
-					<a href="{{.BaseRepo.Link}}/compare/{{.BaseRepo.DefaultBranch}}...{{.Username}}:{{.BranchName}}">
+					<a href="{{.BaseRepo.Link}}/compare/{{.BaseRepo.DefaultBranch}}...{{.BranchName}}">
```

Comment:
But on compare UI, it's not ready for compare between forks. I think this need another PR to do that. For this one, we should let the button works at first.
Gitea's vs Github's
![untitled](https://cloud.githubusercontent.com/assets/81045/23494484/14a87e3e-ff4e-11e6-9f85-3baa75ca9d1e.png)